### PR TITLE
Bumped sensor sdk version to 1.3.0 and body tracking to 1.0.0

### PR DESCRIPTION
**Pull request by RoseFlunder**
### Description of the changes:
- Changed the version numbers to the current versions for the sensor and body tracking sdk

### Before submitting a Pull Request:
- [ ] I reviewed [CONTRIBUTING.md](https://github.com/microsoft/Azure_Kinect_ROS_Driver/blob/master/CONTRIBUTING.md)
- [ ] I [built my changes](https://github.com/microsoft/Azure_Kinect_ROS_Driver/blob/master/docs/building.md) locally
- [ ] I tested my changes with a device

### I tested changes on: 
- [ ] Windows
- [x] Linux

Manual testing with a recording file.

### CODE REVIEW

**Comment by ooeygui** [comment]
Thank you for the contribution. Please update the Windows build as well.

**Comment by RoseFlunder** [comment]
I don't have a ROS on windows installation at hand but when looking at the cmake files I would expect it to work with windows as well.
Do you see something that should be updated?

The pipeline for windows seems to work as well.

**Comment by ooeygui** [comment]
The Windows SDK in the pipeline is 1.2. Could you update that to the correct 1.3 SDK?

```
# Download and install the Azure Kinect Sensor SDK 
  - powershell: |
      wget http://download.microsoft.com/download/1/9/8/198048e8-63f2-45c6-8f96-1fd541d1b4bc/Azure%20Kinect%20SDK%201.2.0.msi -OutFile $(Build.SourcesDirectory)\sdk.msi
      $(Build.SourcesDirectory)\sdk.msi /passive
```

**Comment by RoseFlunder** [comment]
Oh, already had this in my Editor but wasn't saved before the commit.
Thanks for the hint.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ jobs:
 
   # Download and install the Azure Kinect Sensor SDK 
   - powershell: |
-      wget http://download.microsoft.com/download/1/9/8/198048e8-63f2-45c6-8f96-1fd541d1b4bc/Azure%20Kinect%20SDK%201.2.0.msi -OutFile $(Build.SourcesDirectory)\sdk.msi
+      wget https://download.microsoft.com/download/e/6/6/e66482b2-b6c1-4e34-bfee-95294163fc40/Azure%20Kinect%20SDK%201.3.0.exe -OutFile $(Build.SourcesDirectory)\sdk.msi
```
Comment (ooeygui) [inline azure-pipelines.yml]: This code is downloading a .exe, and renaming it to .msi, then attempting to run the msi with an msi command line parameter. the /passive means don't display UI, which with an exe will fail and block the build.

Could you change to the msi associated with 1.3.0?

**Comment by RoseFlunder** [comment]
@ooeygui
Is everything resolved now or are additional changes required?

File: azure-pipelines.yml (hunk 1 above)
Comment (ooeygui) [reply azure-pipelines.yml]: Sorry, I this was a pending comment :-(

Comment (RoseFlunder) [reply azure-pipelines.yml]: Hmm I can't find a downloadlink for a msi package.
On github is a msi link stated but it links the the exe file:
[https://github.com/microsoft/Azure-Kinect-Sensor-SDK/issues/800](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/issues/800)

In the Azure kinect docs website is also only the .exe file linked as "Microsoft Installer"
[https://docs.microsoft.com/en-us/azure/kinect-dk/sensor-sdk-download](https://docs.microsoft.com/en-us/azure/kinect-dk/sensor-sdk-download)

Also in the usage guide in the github repo they only link an .exe:
https://github.com/microsoft/Azure-Kinect-Sensor-SDK/blob/develop/docs/usage.md

Comment (RoseFlunder) [reply azure-pipelines.yml]: Will it work if we just name the output file sdk.exe?
The /passive seems to work with it too for me.
I will get the Windows UAC popup on my windows desktop machine but after that it installs the sdk in the background.
I guess the pipeline doesn't have that.

Comment (RoseFlunder) [reply azure-pipelines.yml]: I just pushed the change for testing. Could you check if it works?
I don't know where to look in the pipeline exactly.

Comment (ooeygui) [reply azure-pipelines.yml]: I'm following up with the product manager for the kinect.

**Review by ooeygui — APPROVED** [review]

**Pull request merged**